osu.Desktop/Overlays/VersionManager.cs

```diff
@@ -222,44 +226,5 @@ protected override void PopOut()
         {
         }
 
-        private class UpdateProgressNotification : ProgressNotification

```

Comment:
this was much more contained and nicer to look at.

Reply:
I do not know what happened with container there. 

So, should i create factory~ish functions  for creating the notifications?

Reply:
i don't have a ready solution, just stating i still prefer the old code in the current state.